p2p_wallet/Extensions/PropertyWrappers/UserDefaults.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+@propertyWrapper
+struct UserDefault<Value> {
```

Comment:
Why do we need UserDefault property wrapper when SwiftyUserDefaults has already handle this?